src/main/scala/co/datamechanics/delight/Configs.scala

```diff
@@ -51,7 +51,10 @@ object Configs {
   }
 
   def generateDMAppId(sparkConf: SparkConf): String = {
-    val appName: String = sparkConf.get("spark.delight.appNameOverride", sparkConf.get("spark.app.name", "undefined"))
+    val appName: String = sparkConf.getOption("spark.delight.appNameOverride")
```

Comment:
nit: Are these `String` types for this and the following vals required? Looks a bit odd.

Reply:
Addressed in this fixup commit 1b0410cb8601cc725ef5255c7a153320d81a144a thanks!